pkg/node/node.go

```diff
@@ -109,6 +112,19 @@ func NewBee(o Options) (*Bee, error) {
 		logger.Infof("new libp2p key created")
 	}
 
+	var stateStore storage.StateStorer
+	if o.DataDir == "" {
+		stateStore = mockinmem.NewStateStore()
+		logger.Warning("using in-mem state store. no node state will be persisted")
+	} else {
+		stateStore, err = leveldb.NewStateStore(filepath.Join(o.DataDir, "statestore.db"))
```

Comment:
Sorry for taking too long to respond to the question about state store data location. I see that you already set it here. Which is fine. I would just remove the `.db` suffix as this is a directory in leveldb case.